tests/Stripe/Service/WebhookEndpointServiceTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Stripe\Service;
+
+/**
+ * @internal
+ * @covers \Stripe\Service\WebhookEndpointService
+ */
+final class WebhookEndpointServiceTest extends \PHPUnit\Framework\TestCase
+{
+    use \Stripe\TestHelper;
+
+    const TEST_RESOURCE_ID = 'we_123';
+
+    /** @var \Stripe\StripeClient */
+    private $client;
+
+    /** @var WebhookEndpointService */
+    private $service;
+
+    /**
+     * @before
+     */
+    protected function setUpService()
+    {
+        $this->client = new \Stripe\StripeClient(['api_key' => 'sk_test_123', 'api_base' => MOCK_URL]);
+        $this->service = new WebhookEndpointService($this->client);
```

Comment:
oo which don't you use the lazy loading in tests and do $this->client->webhookEndpoints->xxxx`

Reply:
Mostly because I wrote the individual service tests before I added the service accessors on `StripeClient`.

Reply:
Could we add a few? I didn't find specific tests to confirm the accessors all exist properly?